### PR TITLE
Particle system tooltop translation

### PR DESCRIPTION
**Pull request by hana-alice**
Re: cocos-creator/3d-tasks#6361

Changelog:
 * Particle system tooltip translation.

### CODE REVIEW

**Comment by hana-alice** [comment]
found a bug:
image
image

File: editor/i18n/en/localization.js [hunk 1]
```diff
@@ -431,6 +538,40 @@ module.exports = {
...
+        startRotationZ: 'Angle of rotation around Z-axis at beginning',
+        startDelay: 'Time delayed of particle emit after the system start running',
+        startLifetime: 'Life time of the particle',
+        duration: 'Particle duration',
+        loop: 'Loop animation',
+        prewarm: 'Animation plays from the end state of last round(only valid when loop activated)',
+        simulationSpace: 'The particle space do your calculation in',
+        simulationSpeed: 'Update ratio of particles',
+        playOnAwake: 'Play on awake',
+        gravityModifier: 'How much particles get affected by the gravity',
```
Comment (pandamicro) [inline editor/i18n/en/localization.js]: Only available in CPU particle system.

heck other property only available in CPU with @caryliu1999

File: editor/i18n/en/localization.js [hunk 2]
```diff
@@ -431,6 +538,40 @@ module.exports = {
...
+        startRotation3D: 'Set rotation of X, Y and Z respectively if true',
+        startRotationX: 'Angle of rotation around X-axis at beginning',
+        startRotationY: 'Angle of rotation around Y-axis at beginning',
+        startRotationZ: 'Angle of rotation around Z-axis at beginning',
+        startDelay: 'Time delayed of particle emit after the system start running',
+        startLifetime: 'Life time of the particle',
+        duration: 'Particle duration',
+        loop: 'Loop animation',
+        prewarm: 'Animation plays from the end state of last round(only valid when loop activated)',
+        simulationSpace: 'The particle space do your calculation in',
```
Comment (pandamicro) [inline editor/i18n/en/localization.js]: The particle space your calculation is done in

**Review by pandamicro — COMMENTED** [review]
Very nice work, please take a look at the comment

**Pull request merged**